docs/user_guide.md

```diff
@@ -556,76 +531,184 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
 ![browse project](img/clair_ready.png)
 
-**Scanning an image** 
+Until the database has been fully populated, the timestamp is replaced by a warning symbol. When the database is ready, you can scan images individually or scan all images across all projects.
+
+### Scan Individual Images
+
+1. Log in to the Harbor interface with an account that has at least project administrator privileges.
+1. Go to **Projects** and select a project. 
+1. Select the **Scanner** tab.
+
+   The **Scanner** tab shows the details of the scanner that is currently set as the scanner to use for this project.
+   
+   ![Project scanner tab](img/project-scanners.png)
+1. Click **Edit** to select a different scanner from the list of scanners that are connected to this Harbor instance, and click **OK**.
+
+   ![Project scanner tab](img/select-scanner.png)
+1. Select the **Repositories** tab and select a repository.
+
+   For each tag in the repository, the **Vulnerabilities** column displays the vulnerability scanning status and related information.
+   
+   ![Tag vulnerability status](img/tag-vulnerability-status.png)
+1. Select a tag, or use the check box at the top to select all tags in the repository, and click the **Scan** button to run the vulnerability scan on this image.
+
+   ![Scan an image](img/scan_image.png)
+
+   **NOTE**: You can start a scan at any time, unless the status is **Queued** or **Scanning**. If the database has not been fully populated, there is a warning message at the footer of the **Projects** > **Repositories** view and you cannot run the scan. The following statuses are displayed in the **Vulnerabilities** column:
+   
+   * **Not Scanned:** The tag has never been scanned.
+   * **Queued:** The scanning task is scheduled but has not run yet.
+   * **Scanning:** The scanning task is in progress and a progress bar is displayed.
+   * **View log:** The scanning task failed to complete. Click **View Log** link to view the related logs.
+   * **Complete:** The scanning task completed successfully.
+
+   If the process completes successfully, the result indicates the number of vulnerabilities with each severity level and the number of vulnerabilities that are fixable.
+
+   ![Scan result](img/scan-result.png)
+
+   * **Red:** Critical level of vulnerabilities
+   * **Orange:** High level of vulnerabilities
+   * **Yellow:** Medium level of vulnerabilities
+   * **Blue:** Low level of vulnerabilities
+   * **Green:** No vulnerability
+   * **Grey:** Unknown level of vulnerabilities
+1. Hover over the number of fixable vulnerabilities to see a summary of the vulnerability report. 
+
+   ![Vulnerability summary](img/vulnerability-summary.png)
+1. Click on the tag name to see a detailed vulnerability report.
+ 
+  ![Vulnerability report](img/tag_detail.png)
+  
+   In addition to information about the tag, all of the vulnerabilities found in the last scan are listed. You can order or filter the list by the different columns.
```

Comment:
The user can also issue a scan process in this detail page if they want

Reply:
Added in https://github.com/goharbor/harbor/pull/10029/commits/a1eac99a49aa192cb5048c5eaa8a4e2a756cd134.